bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapJSon.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.shelly.internal.coap;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.commons.lang.Validate;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * The {@link ShellyCoapJSon} helps the CoIoT Json into Java objects
+ *
+ * @author Markus Michels - Initial contribution
+ */
+public class ShellyCoapJSon {
+
+    public static final String CoIoT_Tag_blk = "blk";
+    public static final String CoIoT_Tag_sen = "sen";
+    public static final String CoIoT_Tag_act = "act";
+
+    public static final String CoIoT_Tag_ID = "I";
+    public static final String CoIoT_Tag_DESC = "D";
+    public static final String CoIoT_Tag_TYPE = "T";
+    public static final String CoIoT_Tag_RANGE = "R";
+    public static final String CoIoT_Tag_LINKS = "L";
+
+    public static final String CoIoT_Tag_Generic = "G";
+
+    public static class CoIoT_Descr_blk {
+        String I; // ID
```

Comment:
Use the Java naming conventions for fields. If the json contains another name use `@SerializedName`:
```suggestion
        @SerializedName("I")
        String id;
```

Reply:
done